invideoquiz/tests/test_invideoquiz_view.py

```diff
@@ -0,0 +1,14 @@
+from xblock.test.tools import TestRuntime as Runtime
+from xblock.runtime import DictKeyValueStore, KvsFieldData
+from invideoquiz import InVideoQuizXBlock
+from unittest.mock import Mock
+
+
+def test_invideoquiz_view():
+    key_store = DictKeyValueStore()
+    field_data = KvsFieldData(key_store)
+    runtime = Runtime(services={'field-data': field_data})
+    instace = InVideoQuizXBlock(runtime, scope_ids=Mock())
```

Comment:
nit: could this variable be named `instance`?

Reply:
I'd second this request, and even call it _not_ a "nit"; misspelled words/variables should always be corrected ("nit" implies an optional request)